package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
> Adding new lint rules could easily be not backwards compatible for some projects.

Again: I consider that backwards compatible. Whether or not your code passes the linter is not part of the public API. Even micro bumps can break that (when they fix broken rules), so...

> I was going to suggest Greenkeeper

It's good to avoid upper bounds, when they are not strictly necessary (and they are often not for build tools).